### ISSUE TITLE
Installing dbt-snowflake 0.19.0rc1 on Python 3.9.1 (Windows) failes

### ISSUE THREAD

**Issue by swiffer** [fishtown-analytics/dbt#2985]
### Describe the bug
Installing dbt-snowflake 0.19.0rc1 on Windows currently fails with:

```bash
No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
      error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
```

Looks like there are no prebuilt wheels for Python 3.9 for the numpy version required. Works fine in 3.8.

They are available starting from 1.19.3: https://github.com/numpy/numpy/releases/tag/v1.19.3

**The operating system you're using:** Windows 10 20H2

**The output of `python --version`:** Python 3.9.1

**Comment by jtcohen6** [comment]
Thanks for the report @swiffer. We aren't yet able to support python 3.9 with `dbt-snowflake` because of a few conflicts  with dependencies of `snowflake-connector-python`. We'll be able to revisit this one for a future release, when we can declare "official" py39 compatibility across the board.

(cc @kwigley)

**Comment by kwigley** [comment]
should be resolved by #3148!
